### PR TITLE
Add 3.7.fp7 release notes.

### PR DESCRIPTION
**Pull request by segalaj**

### CODE REVIEW

File: admin/en/source/pages/changelog.rst [hunk 1]
```diff
@@ -11,13 +11,44 @@ Release Date: 2017-09-04
 New Features
 ~~~~~~~~~~~~
 * Fujitsu Cloud Service K5 US, Finland and Spain regions are supported.
-* Can create images in PXE format for CentOS.
-* Users can deploy instances from published machine images to OpenStack.
 * Introduce a REST API for users to list files to synchronize to the target environment after a CentOS scan without overlay.
+* Users can deploy instances from published machine images to OpenStack.
+* Can create images in PXE format for CentOS.
```
Comment (emuus) [inline admin/en/source/pages/changelog.rst]: I would rewrite as : Images can be created in PXE format for CentOS

File: apis/en/source/pages/changelog.rst [hunk 2]
```diff
@@ -11,13 +11,44 @@ Release Date: 2017-09-04
 New Features
 ~~~~~~~~~~~~
 * Fujitsu Cloud Service K5 US, Finland and Spain regions are supported.
-* Can create images in PXE format for CentOS.
-* Users can deploy instances from published machine images to OpenStack.
 * Introduce a REST API for users to list files to synchronize to the target environment after a CentOS scan without overlay.
+* Users can deploy instances from published machine images to OpenStack.
+* Can create images in PXE format for CentOS.
```
Comment (emuus) [inline apis/en/source/pages/changelog.rst]: see above

File: end-user/en/source/pages/changelog.rst [hunk 3]
```diff
@@ -11,13 +11,44 @@ Release Date: 2017-09-04
 New Features
 ~~~~~~~~~~~~
 * Fujitsu Cloud Service K5 US, Finland and Spain regions are supported.
-* Can create images in PXE format for CentOS.
-* Users can deploy instances from published machine images to OpenStack.
 * Introduce a REST API for users to list files to synchronize to the target environment after a CentOS scan without overlay.
+* Users can deploy instances from published machine images to OpenStack.
+* Can create images in PXE format for CentOS.
```
Comment (emuus) [inline end-user/en/source/pages/changelog.rst]: see above rewrite

File: admin/en/source/pages/changelog.rst [hunk 4]
```diff
@@ -11,13 +11,55 @@ Release Date: 2017-09-04
...
+* 6501 Deleted package files are recovered after migration
+* 6507 Debian migration cannot detect missing info changes
+* 6573 "Uploading Archive 0 %" and "Transfer in progress -1 %" are displayed alternately by template import
+* 6611 Error message when trying to delete a UForge OS profile milestone not self-explanatory
+* 6614 UForge web service response (401 unauthorized) is not RFC compliant
+* 6683 Heap memory error when scanning VM with huge files number
+* 6753 AWS publish failed in some circumstances
+* 6760 uforge-install-config TUI does not appear when using K5 Console
+* 6768 Unable to build a package using rpmgen with a file located in /
+* 6799 Instances on AWS do not pass status check anymore: status check 1/2 and impossibility to connect to the instance
```
Comment (jbremond) [inline admin/en/source/pages/changelog.rst]: I think this commit was not inside the release. Should be removed from the RN

**Pull request merged**